src/main/java/org/tndata/android/compass/ui/PriorityItemView.java

```diff
@@ -9,34 +9,61 @@
 import android.widget.LinearLayout;
 import android.widget.TextView;
 
+import org.tndata.android.compass.adapter.MyPrioritiesGoalAdapter;
+
 
 /**
- * Created by isma on 7/17/15.
+ * A view containing a priority item in a linear layout.
+ *
+ * @author Ismael Alonso
+ * @version 1.0.0
  */
 public class PriorityItemView extends LinearLayout{
-    private Context mContext;
-
     private ImageView mImageView;
     private TextView mTextView;
 
+    private MyPrioritiesGoalAdapter.ItemHierarchy mItemHierarchy;
 
+
+    /**
+     * Constructor.
+     *
+     * @param context the context.
+     */
     public PriorityItemView(Context context){
         this(context, null, 0);
     }
 
-    public PriorityItemView(Context context, AttributeSet attrs) {
+    /**
+     * Constructor.
+     *
+     * @param context the context.
+     * @param attrs the attribute set.
+     */
+    public PriorityItemView(Context context, AttributeSet attrs){
         this(context, attrs, 0);
     }
 
-    public PriorityItemView(Context context, AttributeSet attrs, int defStyle) {
+    /**
+     * Constructor.
+     *
+     *
+     * @param context the context.
+     * @param attrs the attribute set.
+     * @param defStyle the style resource.
+     */
+    public PriorityItemView(Context context, AttributeSet attrs, int defStyle){
         super(context, attrs, defStyle);
         initViews();
     }
 
+    /**
+     * Creates the layout of the view.
+     */
     private void initViews(){
         setOrientation(LinearLayout.HORIZONTAL);
-        setLayoutParams(new ViewGroup.LayoutParams(ViewGroup.LayoutParams.WRAP_CONTENT,
-                ViewGroup.LayoutParams.WRAP_CONTENT));
+        setLayoutParams(new ViewGroup.LayoutParams(ViewGroup.LayoutParams.MATCH_PARENT,
+                getPixels(35)));
```

Comment:
Will this be OK across different screen sizes? Should we put the `35` in `dimens.xml`?